edx_event_bus_kafka/internal/consumer.py

```diff
@@ -274,6 +286,25 @@ def reset_offsets(consumer, partitions):
         finally:
             self.consumer.close()
 
+    def consume_indefinitely(self, offset_timestamp=None):
+        """
+        Consume events from a topic in an infinite loop.
+
+        Or, if timestamp supplied, call ``reset_offsets_indefinitely`` instead and do not consume.
+        This option is deprecated, and that method should be called directly instead.
+        """
+        # TODO: Once this deprecated argument can be removed, just
+        # remove this delegation method entirely and rename
+        # `_consume_indefinitely` to no longer have the `_` prefix.
+        if offset_timestamp is None:
+            self._consume_indefinitely()
+        else:
+            warnings.warn(
+                "Calling consume_indefinitely with offset_timestamp is deprecated; "
+                "please call reset_offsets_indefinitely directly instead."
+            )
```

Comment:
Author's note: This warning will fire for the tests because I haven't changed most of them to use the new method directly; this is to retain coverage.